client/app/components/packages/rwcds-form/show.hbs

```diff
@@ -109,39 +109,48 @@
             'label'
             affectedZoningResolution.dcpZoningresolutiontype)
           as |zrTypeLabel|}}
-          <div
-            class="ruled-adjacent grid-x"
-          >
-            <div
-              class="text-weight-bold cell auto large-padding-right"
-            >
-              {{~zrTypeLabel~}}
-              {{#if (lookup-action-type zrTypeLabel)}}
-                <LabsUi::IconTooltip
-                  @tip={{lookup-action-type zrTypeLabel}}
-                  @icon="info-circle"
-                  @side="bottom"
-                  @transform="shrink-4 up-2"
-                  @fixedWidth={{true}}
-                />
-              {{/if}}
-              {{affectedZoningResolution.dcpZrsectiontitle}}
-            </div>
-            <div
-              class="cell large-3"
-            >
+
+          <h3 class="large-margin-top small-margin-bottom">{{zrTypeLabel}}</h3>
+
+          <Ui::Answer @beside={{true}} as |A|>
+            <A.Prompt>
+              Modifying Zoning Resolution sections:
+            </A.Prompt>
+            <A.Field>
               {{affectedZoningResolution.dcpModifiedzrsectionnumber}}
-              {{#if (eq affectedZoningResolution.dcpZoningresolutiontype (optionset 'affectedZoningResolution' 'actions' 'code' 'ZR'))}}
+            </A.Field>
+          </Ui::Answer>
+
+          <Ui::Answer @beside={{true}} as |A|>
+            <A.Prompt>
+              Pursuant to Zoning Resolution section:
+            </A.Prompt>
+            <A.Field>
+              {{affectedZoningResolution.dcpZrsectiontitle}}
+            </A.Field>
+          </Ui::Answer>
+
```

Comment:
Can you summarize what's going on in the diff here? `Ui::Answer` wasn't used before.